msgraph/valuetypes.go

```diff
@@ -22,10 +22,11 @@ func (s StringNullWhenEmpty) MarshalJSON() ([]byte, error) {
 	return json.Marshal(string(s))
 }
 
-type AccessPackageCatalogStatus = string
+type AccessPackageCatalogState = string
```

Comment:
Per compatibility note, can we keep both these types until such time as the beta API is updated